src/clr-base/src/lib/utils/helpers/exists.spec.ts

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (c) 2016-2019 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import { exists } from './exists';
+
+describe('Functional Helper: ', () => {
+  describe('exists() ', () => {
+    beforeEach(function() {
+      (window as any).test = {
+        levelOne: {
+          levelTwo: {
+            levelThree: {
+              notTrue: false,
+              notStrung: '',
```

Comment:
`notStrung`? intentional or a typo for notString?

Reply:
Having fun with unit tests means writing unit tests is fun! It's intentional.